app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -54,6 +54,10 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
     }
   }
 
+  private fun backClicked(): Next<PatientSummaryModel, PatientSummaryEffect> {
+    return dispatch(GoBackToPreviousScreen)
```

Comment:
The commit message includes the qualifier of the patient being dead, but it doesn't seem like we are checking the patient status in this commit. Should we reword the message?

Reply:
Ah, I forgot I can just return `noChange`.  Making that change.